.github/workflows/build-test-publish.yml

```diff
@@ -88,7 +88,7 @@ jobs:
     - uses: actions/checkout@v4
     - uses: actions/cache@v4
       with:
-        path: /var/lib/apt
+        path: ${{ runner.temp }}/cache-linux
```

Comment:
`Warning: Path Validation Error: Path(s) specified in the action for caching do(es) not exist, hence no cache is being saved.`

Reply:
I'm fine if we want to drop this; I do not have a strong opinion. Let me know if you firmly believe we should go ahead and remove it, or whether you want other maintainers to weigh in.